frontend/packages/noobaa-storage-plugin/integration-tests/tests/backingStoreTest.scenario.ts

```diff
@@ -0,0 +1,64 @@
+import { execSync } from 'child_process';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import { $, ExpectedConditions as until, browser } from 'protractor';
+import { appHost, testName } from '@console/internal-integration-tests/protractor.conf';
+import {
+  operatorsPage,
+  ocsOperator,
+  BackingStoreHandler,
+  Providers,
+  createBackingStoreLink,
+} from '../views/createBS.view';
+
+const NAME_BS = `${testName}-bs`;
+const NS = 'openshift-storage';
+
+const bsHandler = new BackingStoreHandler(NAME_BS);
+
+describe('Tests creation of Backing Store', () => {
+  beforeAll(async () => {
+    await browser.get(`${appHost}/ns/${NS}`);
+  });
+
+  beforeEach(async () => {
+    await browser.wait(until.and(crudView.untilNoLoadersPresent));
+    await operatorsPage();
+    await browser.wait(until.and(crudView.untilNoLoadersPresent));
+    await browser.wait(until.visibilityOf(ocsOperator));
+    await ocsOperator.click();
+    await browser.wait(until.and(crudView.untilNoLoadersPresent));
+    await browser.wait(until.visibilityOf(createBackingStoreLink));
+    await createBackingStoreLink.click();
+    await browser.wait(until.and(crudView.untilNoLoadersPresent));
+  });
+
+  it('Tests creation of AWS S3 backing store', async () => {
+    const { name } = await bsHandler.createStore(Providers.AWS);
+    expect(name).toEqual(NAME_BS);
+    execSync(`kubectl delete secrets ${NAME_BS}-secret -n openshift-storage`);
+  });
+
+  it('Tests creation of Azure backing store', async () => {
+    const { name } = await bsHandler.createStore(Providers.AZURE);
+    expect(name).toEqual(NAME_BS);
+    execSync(`kubectl delete secrets ${NAME_BS}-secret -n openshift-storage`);
+  });
+
+  it('Tests creation of S3 Endpoint Type', async () => {
+    const { name } = await bsHandler.createStore(Providers.S3);
+    expect(name).toEqual(NAME_BS);
+    execSync(`kubectl delete secrets ${NAME_BS}-secret -n openshift-storage`);
+  });
+
+  it('Tests creation of PVC Endpoint Type', async () => {
+    const { name } = await bsHandler.createStore(Providers.PVC);
+    expect(name).toEqual(NAME_BS);
+  });
+
+  afterEach(async () => {
```

Comment:
Before/after functions are better to define at the top of the respective describe block for visibility.